packages/terra-application/src/terra-dev-site/app/components.2/ApplicationNavigation.app.mdx

```diff
@@ -255,6 +255,24 @@ to render the action controls necessary for its usage. Failure to do so will lea
 
 </Notice>
 
+### `userActionConfig`
+#### Is Required: `false`
+
+The `userActionConfig` prop allows consumers have first class support for a action utility button.
+If the `userActionConfig` prop is not set a action utility button will not be displayed.
+
+|Key Name|Type|Is Required|Description|
+|---|---|---|---|
+|`text`|String|**required**|Text to be displayed on user action button..
+|`userActionCallback`|Function|**required**|A function to be executed on user action button click.|
+
+```jsx
+const userActionConfig = {
+  text: 'Edit Photo',
+  userActionCallback: () => console.log('Button Clicked'),
+};
+```
```

Comment:
Lets move this higher in the doc page so it is immediately right after the`userConfig` section and before `hero`, so line 215. Plus small edits to the doc:

```suggestion
### `userActionConfig`
#### Is Required: `false`

The `userActionConfig` prop allows consumers to add an additional item to the `userConfig` section where it is displayed within the application. If a specific action related to a User's information or details, consumers can use this prop which will add a button along with the user name, detail, and avatar. Note: if the `userConfig` prop is not set, the user action button will not be displayed.

|Key Name|Type|Is Required|Description|
|---|---|---|---|
|`text`|String|**required**|Text to be displayed on user action button.
|`userActionCallback`|Function|**required**|A function to be executed on user action button click.|

```jsx
const userActionConfig = {
  text: 'User Specific Action',
  userActionCallback: () => console.log('Button Clicked'),
};
```

\
\
Also -- lets make an update to line [209](https://github.com/cerner/terra-application/blob/5853c1b76c80abb6a024252b4d35ab24ca1f5686/packages/terra-application/src/terra-dev-site/app/components.2/ApplicationNavigation.app.mdx?plain=1#L209) from:
```
  detail: 'Is a User',
```
to: 
```
  detail: 'DomainName123 (or) UserRole',
```